pyquil/wavefunction.py

```diff
@@ -100,6 +100,30 @@ def ground(cls, qubit_num):
         container[0] = 1.0
         return cls(container)
 
+    def plot(self, qubit_subset=None):
+        """
+        Plots a bar chart with bitstring on the x axis and probability on the y axis.
+
+        :param list qubit_subset: Optional parameter used for plotting a subset of the Hilbert space.
+        """
+        import matplotlib.pyplot as plt
+        prob_dict = self.get_outcome_probs()
+        if not qubit_subset:
+            plt.bar(range(len(prob_dict)), prob_dict.values(), align='center')
```

Comment:
These 3 lines and 123-125 are the same with the exception of which dict is used. Can you refactor to set prob_dict depending on the value of qubit_subset?

Reply:
These 3 lines = 112-115